app/services/search/strategies/elasticsearch.rb

```diff
@@ -0,0 +1,45 @@
+class Search::Strategies::ElasticSearch
+  def initialize(search_params)
+    @keyword = search_params[:keyword]
+    @coordinates = search_params[:coordinates]
+    @radius = search_params[:radius]
+    @polygons = search_params[:polygons]
+    @filters = search_params[:filters]
+    @replica = search_params[:replica]
+    @per_page = search_params[:per_page]
+    @page = search_params[:page]
+    @typo_tolerance = search_params[:typo_tolerance]
+  end
+
+  def vacancies
+    @vacancies ||= Vacancy.includes(:organisations).search(@keyword, search_arguments)
+  rescue ElasticSearch::ElasticSearchProtocolError => e
+    Rollbar.error("ElasticSearch search error", details: e, search_arguments: search_arguments)
+    @vacancies = Vacancy.none
+  end
+
+  def total_count
+    return 0 if vacancies.none?
+
+    vacancies.raw_answer["nbHits"]
+  end
+
+  private
+
+  def big_if_true(bool)
+    bool ? "big" : "small"
```

Comment:
🚫 🙅 

Are you sure this is correct? 
What's big? What's small?
Can you extract this in well defined global constants?

Reply:
@DFE-Digital/teaching-vacancies-developers discuss!

Reply:
Happy to use global variables if you prefer?